x/incentives/types/gov.go

```diff
@@ -0,0 +1,73 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+const (
+	ProposalTypeCreateGaugeGroups = "CreateGaugeGroups"
+)
+
+func init() {
+	govtypes.RegisterProposalType(ProposalTypeCreateGaugeGroups)
+	govtypes.RegisterProposalTypeCodec(&CreateGaugeGroupsProposal{}, "osmosis/CreateGaugeGroupsProposal")
+}
+
+var (
+	_ govtypes.Content = &CreateGaugeGroupsProposal{}
+)
+
+// NewCreateGaugeGroupsProposal returns a new instance of a gauge group creation proposal struct.
+func NewCreateGaugeGroupsProposal(title, description string, groups []CreateGroup) govtypes.Content {
+	return &CreateGaugeGroupsProposal{
+		Title:        title,
+		Description:  description,
+		CreateGroups: groups,
+	}
+}
+
+func (p *CreateGaugeGroupsProposal) GetTitle() string { return p.Title }
+
+// GetDescription gets the description of the proposal
+func (p *CreateGaugeGroupsProposal) GetDescription() string { return p.Description }
+
+// ProposalRoute returns the router key for the proposal
+func (p *CreateGaugeGroupsProposal) ProposalRoute() string { return RouterKey }
+
+// ProposalType returns the type of the proposal
+func (p *CreateGaugeGroupsProposal) ProposalType() string {
+	return ProposalTypeCreateGaugeGroups
+}
+
+// ValidateBasic validates a governance proposal's abstract and basic contents
+func (p *CreateGaugeGroupsProposal) ValidateBasic() error {
+	if len(p.CreateGroups) == 0 {
+		return fmt.Errorf("must provide at least one gauge group")
```

Comment:
These should be pool IDs, not gauge groups. Related to another comment of mine that claims we should have pool IDs be directly embedded into the proposal struct as opposed to having these `CreateGroups` structs

Reply:
Struct is needed as explained above